dh/sidh/doc.go

```diff
@@ -1,30 +1,44 @@
-// Package sidh provides implementation of experimental post-quantum
+// Package sidh is deprecated, it provides SIDH and SIKE key encapsulation
+// mechanisms.
+//
+// DEPRECATION NOTICE
+//
+// SIDH and SIKE are deprecated as were shown vulnerable to a key recovery
+// attack by Castryck-Decru's paper (https://eprint.iacr.org/2022/975). New
+// systems should not rely on this package. This package is frozen.
```

Comment:
By "frozen" I suppose you mean that maintenance has stopped and bug fixes won't be considered? Consider saying this explicitly.

Reply:
It's frozen in the sense that it's preserved for historical reasons and to indicate people should not use it to secure any system. 
Bugs can still appear at anytime in software for many different reasons.

Reply:
Also there is still code implementing field arithmetic which can be (re)used for other purposes.